### PR TITLE
Move eslint ignores into .eslintrc

### PR DESCRIPTION
**Pull request by RedGuy12**
Resolves lots of files in root

### Changes

updates the eslint ignore patterns, move them out of root, add root: true

### Reason for changes

too many files in root
### Tests

it works

### CODE REVIEW

**Comment by WorldLanguages** [comment]
VS Code now has file nesting. By adding a `.vscode` folder, we can now achieve this:

image

It looks like this collapsed:
image

File: .eslintrc.json [hunk 1]
```diff
@@ -1,4 +1,6 @@
 {
+  "root": true,
```
Comment (RedGuy12) [inline .eslintrc.json:2]: this is best practice so when linting it doesn't inherit any personal configs

**Comment by RedGuy12** [comment]
@WorldLanguages can i/you merge this pls
this is tiny and has no downsides

File: .eslintrc.json (hunk 1 above)
Comment (WorldLanguages) [reply .eslintrc.json:2]: Good idea!

**Pull request merged**